exercises/concept/arcade-high-score/.meta/design.md

```diff
@@ -0,0 +1,35 @@
+# Design
+
+## Learning objectives
+
+- Know what a map is.
+- Know how to define a map literal.
+- Know how to add key/value pairs in a map.
+- Know how to get values from a map.
+- Know how to get keys from a map.
+- Know how to update values in a map.
+- Know how to remove items from a map.
+- Know how to retrieve all items in a map.
+- Know what a `HashMap` is.
+- Know what a `LinkedHashMap` is.
+- Know what a `TreeMap` is.
+- Know the main differences in `HashMap`, `LinkedHashMap` and `TreeMap`.
```

Comment:
Note on these objectives: they should cover the learning objectives from the _exercise_, not the _concept_. I believe this exercise doesn't cover the different implementations of the `Map` interface, nor how to retrieve all items in a map.